.gitignore

```
@@ -1,5 +1,6 @@
 dist
 node_modules
+.env
```

Sorting the entries there

```suggestion
.env
dist
node_modules
```